addons/website_blog/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, "website_blog", "migrations/13.0.1.0/noupdate_changes.xml")
```

Comment:
This is wrong. You have to remove `name` and `subtitle` from that file:

https://github.com/OCA/OpenUpgrade/blob/1acf10ec2cf2839bd3ff3fa066142143a93ca86b/addons/website_blog/migrations/13.0.1.0/noupdate_changes.xml#L4-L5

Reply:
The fix is in https://github.com/OCA/OpenUpgrade/pull/2418.